app/controllers/application_controller.rb

```diff
@@ -70,4 +72,10 @@ def set_no_cache_header
   def touch_current_user
     current_user.touch
   end
+
+  private
+
+  def core_pages?
```

Comment:
@rhymes this method common for alll controllers

Reply:
Sorry, don't know how I missed it